reset_react.sh

```diff
@@ -1,5 +1,4 @@
 rm -rf node_modules
-rm package-lock.json
```

Comment:
unsure anyone should be running `reset-react.sh` anymore... thoughts? it updates all deps to latest versions and if this is done on en mass, could cause unknown issues later.


Suggest instead we use dependabot to manage deps

Reply:
Yeah I feel like it's kind of a sore thumb having it in the repo. I'm okay to remove it

Reply:
kill it.